src/test/java/org/opensearch/ad/AnomalyDetectorRestTestCase.java

```diff
@@ -150,6 +150,11 @@ protected AnomalyDetector createAnomalyDetector(AnomalyDetector detector, Boolea
             }
         } while (i < MAX_RETRY_TIMES);
         assertNotNull("Can't get anomaly detector from index", detectorInIndex);
+        try {
+            Thread.sleep(2500);
```

Comment:
nit: can you add comment above this about why it's being added?

Reply:
done